corehq/apps/zapier/consts.py

```diff
@@ -1,3 +1,15 @@
+from corehq.motech.repeaters.models import UpdateCaseRepeater, CreateCaseRepeater, CaseRepeater
+
+
 class EventTypes(object):
     NEW_FORM = 'new_form'
     NEW_CASE = 'new_case'
+    UPDATE_CASE = 'update_case'
+    CHANGED_CASE = 'changed_case'  # includes both new and updated
```

Comment:
Don't love these two names. Couldn't think of a good way to differentiate including the creation as an update and not. Not sure how important it is as long as it's clear in code.